opba-protocols/xs2a-protocol/src/main/resources/processes/xs2a/authorization/consent/xs2a-authorize-consent.bpmn20.xml

```diff
@@ -88,6 +88,40 @@
     <sequenceFlow id="noOauth2Authorization" sourceRef="sid-28d87700-36b4-4ef7-9f81-42473767bb7e" targetRef="startAuthorization">
       <conditionExpression xsi:type="tFormalExpression"/>
     </sequenceFlow>
+    <exclusiveGateway id="isDecoupledScaSelected" name="Is the selected SCA decoupled" default="nonDecoupledScaSelected"/>
+    <sequenceFlow id="nonDecoupledScaSelected" sourceRef="isDecoupledScaSelected" targetRef="embeddedScaChallenge">
+      <conditionExpression xsi:type="tFormalExpression"/>
+    </sequenceFlow>
+    <subProcess id="decoupledScaStatusChecking" flowable:exclusive="true" name="Decoupled Sca status checking subprocess">
+      <startEvent id="decoupledSubProcessStart" name="Start of decoupled Sca subprocess"/>
+      <serviceTask id="checkIfDecoupledScaFinalized" flowable:exclusive="true" name="Checks authorization status to be finalized" flowable:delegateExpression="${xs2aAisCheckDecoupledAuthorizationStatus}"/>
+      <exclusiveGateway id="authrizationSucceded" name="Authirization finalised?"/>
+      <sequenceFlow id="startAuthoriztionChecking" sourceRef="decoupledSubProcessStart" targetRef="checkIfDecoupledScaFinalized" name="Start authoriztion checking"/>
+      <sequenceFlow id="sid-f525dcb0-40f2-4159-a992-729ee3e02130" sourceRef="checkIfDecoupledScaFinalized" targetRef="authrizationSucceded"/>
+      <intermediateCatchEvent id="waitAuthorizationConfirmation" name="Waits until authorization is finished by PSU">
+        <timerEventDefinition>
+          <timeDuration>${flowable.timer.delayDuration}</timeDuration>
+        </timerEventDefinition>
+      </intermediateCatchEvent>
+      <sequenceFlow id="startTimer" sourceRef="authrizationSucceded" targetRef="waitAuthorizationConfirmation" name="Starts timer to wait for PSU actions">
+        <conditionExpression xsi:type="tFormalExpression">#{!xs2aConsentInfo.isDecoupledScaFinalizedByPSU(CONTEXT)}</conditionExpression>
+      </sequenceFlow>
+      <sequenceFlow id="restartAuthorizationCheck" sourceRef="waitAuthorizationConfirmation" targetRef="checkIfDecoupledScaFinalized" name="Starts authorization check after some delay"/>
+      <intermediateCatchEvent id="subprocessTimeLimit" name="Time limit of subprocess execution">
```

Comment:
Please rename to "Decoupled authorization timelimit (timeout)"

Reply:
Renamed for both payment/consent